src/util/antlr/ANTLRErrorHandling.h

```diff
@@ -1,19 +1,31 @@
-// Copyright 2021, University of Freiburg,
-// Chair of Algorithms and Data Structures.
-// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
-
-#pragma once
-
-#include <string>
-
-#include "DefaultErrorStrategy.h"
-#include "RecognitionException.h"
-
-struct ThrowingErrorStrategy : public antlr4::DefaultErrorStrategy {
-  void reportError(antlr4::Parser*,
-                   const antlr4::RecognitionException& e) override {
-    throw antlr4::ParseCancellationException{
-        e.what() + std::string{" at token \""} +
-        e.getOffendingToken()->getText() + '"'};
-  }
-};
+// Copyright 2021, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#pragma once
+
+#include <string>
+
+#include "../../parser/ParseException.h"
+#include "BaseErrorListener.h"
+#include "DefaultErrorStrategy.h"
+#include "RecognitionException.h"
+
+struct ThrowingErrorStrategy : public antlr4::DefaultErrorStrategy {
+  void reportError(antlr4::Parser*,
+                   const antlr4::RecognitionException& e) override {
+    throw antlr4::ParseCancellationException{
+        e.what() + std::string{" at token \""} +
+        e.getOffendingToken()->getText() + '"'};
+  }
+};
+
```

Comment:
This should have a comment (What can this strategy be used for, and what does it do in a nutshell)
In general we should aim for more (concise but useful) docstrings, I'll think of one for the `ErrorStrategy` above.